src/shadertools/modules/picking/picking.js

```diff
@@ -74,27 +74,32 @@ const float COLOR_SCALE = 1. / 256.;
  * Returns highlight color if this item is selected
  * Note: Should be normally be called before other
  */
-vec4 picking_filterHighlight(vec4 color) {
+vec4 picking_filterHighlightColor(vec4 color) {
   bool selected = bool(picking_vRGBcolor_Aselected.a);
   return selected ? picking_uHighlightColor : color;
 }
 
+vec4 picking_filterPickingColor(vec4 color) {
```

Comment:
Add one line comment header. Keep brief as we don't want to generate too long shader strings.